src/main/java/com/derongan/minecraft/mineinabyss/commands/GUICommandExecutor.kt

```diff
@@ -42,6 +45,16 @@ object GUICommandExecutor : IdofrontCommandExecutor() {
                 GondolaGUI(player).show(player)
             }
         }
+        "storage" {
+            playerAction {
+                MIAConfig.data.hubSection?.let {
```

Comment:
I'd prefer 
```kotlin
val hub = MIAConfig.data.hubSection ?: return
```

Reply:
This line creates a compilation error. Anyway `hubSection` is non-null variable now.